### DISCUSSION TITLE
Sending a stream of data from websocket server

### DISCUSSION THREAD

**Discussion by mdagreda** [uNetworking/uWebSockets#1041] [Q&A] [answered]
I'm pretty new to websockets so I'm sorry if my question is dumb. I have read through the user guide and I've been looking through the examples but I'm having trouble understanding how to give the web socket app data and then send it at a rate. I want to be able to send real time updates to a webpage for some data inside my application. If anyone could point me in the direction of where I could find an example of something like this or give me an idea of what I should look into doing that would be very helpful.

**Comment by ghost** [comment, marked as answer]
You can use a timer. This part is not documented, I guess.

Loop::get() returns a Loop* but it is really a us_loop_t* that you can use with uSockets to set up timers
